### ISSUE TITLE
typo fix needed under API in Ioc API.

### ISSUE THREAD

**Issue by Chaitanyassr** [fluid-project/infusion-docs#184]
typo fix needed under API in Ioc API.
<strong>"externalisable"</strong> should be changed.

![Screenshot (258)](https://user-images.githubusercontent.com/30374226/80624399-401c0a00-8a69-11ea-9a53-be3032f4ab72.png)

**Comment by colinbdclark** [comment]
That's the British spelling of the word (with the "s"), which I think is just fine.

**Comment by Chaitanyassr** [comment]
> That's the British spelling of the word (with the "s"), which I think is just fine.

I see. I would be closing down this issue for now then.

**Comment by Chaitanyassr** [comment]
@colinbdclark  I did some research on it. I found that there is a would <strong>externalize</strong>(in British spells as externalise) and there respective noun use that is externalization and externalisation .
there is a word externalizable that is actually an interface in java.io 
but there is no such word as <strong>externalisable</strong>. I am not able to find it in dict. 
@colinbdclark @jobara thoughts?
should i close the issue and PR for now?
